### ISSUE TITLE
[Bug] Minor: Cannot close ImHex while a debug session is running

### ISSUE THREAD

**Issue by cryptax** [WerWolv/ImHex#1153]
### Operating System

Linux

### What's the issue you encountered?

If a debug session is running and you try to close ImHex, a box pops up with an infinite timer saying that there is an ongoing task currently. The box looks like it's waiting for the task to stop, and the only other possibility is to  Cancel this action.

I think we should always be able to close/kill ImHex whatever happens. So, I think the box should either have a button to kill the debugging session, or it should automatically be killed and allow ImHex to be closed

### How can the issue be reproduced?

- Open ImHex
- Load a pattern in the pattern editor
- Put a breakpoint
- Start the debug, wait on the breakpoint
- Attempt to close ImHex

### ImHex Version

1.29.0

### ImHex Build Type

- [X] Nightly or built from sources

### Installation type

AppImage

**Comment by paxcut** [comment]
One thing that hasnt been implemented yet is the set of standard operations that control the execution of sessions that mosdt debuggers have. The debugger is brand new and work in progress. We have the privilege to use it while its being developed and  the features will make their way in due course.
 
In my opinion decisions on how ImHex should behave about quitting when debugging sessions are running should be postponed until the debugger is fully fleshed out and proper execution control of debugging sessions exist. In that light calling this a bug may seem a bit unfair.

**Comment by WerWolv** [comment]
Latest commit should fix this
